tests/unit_tests/test_ecephys.py

```diff
@@ -1,6 +1,14 @@
+from unittest import TestCase
+
+from uuid import uuid4
+from datetime import datetime
+
+import numpy as np
+from pynwb import NWBFile
 from pynwb.misc import Units
+from pynwb.ecephys import ElectricalSeries
 
-from nwbinspector import check_negative_spike_times
+from nwbinspector.checks.ecephys import check_negative_spike_times, check_electrical_series_dims
```

Comment:
All checks can be imported from outermost level; the name of the check ought usually be sufficient for specifying its modality of usage.

Reply:
I'd like to have the option to have checks in different modules share the same name or to have modules that have heavy inits that don't get called automatically